### ISSUE TITLE
Custom runtime-init example is using AWS Secret Manager and DO declaration

### ISSUE THREAD

**Issue by memes** [F5Networks/terraform-gcp-bigip-module#4]
I was going to point a customer to the custom runtime-init example but it contains an AWS onboarding script, not a script that demonstrates onboarding to GCP.

See `examples/bigip_gcp_1nic_deploy_custom_runtime_init/custom_onboard_big.tmpl` for details.

Uses AWS secret manager
```shell
if ${aws_secretmanager_auth}
then
   cat << 'EOF' >> /config/cloud/runtime-init-conf.yaml
  - name: ADMIN_PASS
    type: secret
    secretProvider:
      environment: aws
      type: SecretsManager
      version: AWSCURRENT
      secretId: ${bigip_password}
EOF
else
   cat << 'EOF' >> /config/cloud/runtime-init-conf.yaml
  - name: ADMIN_PASS
    type: static
    value: ${bigip_password}
EOF
fi
```

Uses AWS DO
```yaml
extension_services:
  service_operations:
    - extensionType: do
      type: url
      value: https://raw.githubusercontent.com/f5devcentral/terraform-aws-bigip-module/master/config/onboard_do.json
```

Install as AWS
```shell
bash /var/config/rest/downloads/f5-bigip-runtime-init.gz.run -- '--cloud aws'
```

**Comment by trinaths** [comment]
Created INFRAANO-597 for internal tracking.

**Comment by RavinderReddyF5** [comment]
@memes fixed in v1.1.0
